src/Pages/jihyunglee/Login/Login.js

```diff
@@ -1,13 +1,75 @@
-import React from './node_modules/react';
-import './Login.scss';
+import React from "react";
+import "./Login.scss";
 
 class LoginHyung extends React.Component {
-    render() {
-        return(
-            <div className="LoginHyung">
-            </div>
-        )
+  constructor() {
+    super();
+
+    this.state = {
+      userID: "",
+      userPW: "",
+      isBtnActive: false,
+    };
+  }
+
+  handleInput = (e) => {
+    const { name, value } = e.target;
+    this.setState({ [name]: value }, () => this.canBeSubmitted());
+  };
+
+  canBeSubmitted = () => {
+    const { userID, userPW } = this.state;
+    const isValid = userID.includes("@") > 0 && userPW.length >= 5;
+
+    this.setState({ isBtnActive: isValid });
```

Comment:
object의 key value 이름이 같으면 하나로 줄이실 수 있습니다!
그리고 includes는 boolean으로 나옵니다!
```suggestion
    const isBtnActive = userID.includes("@") && userPW.length >= 5;

    this.setState({ isBtnActive });
```